ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory.tsx

```diff
@@ -116,9 +116,26 @@ function getCountsBySeverity(groups: AlertGroup[]): CountsBySeverity {
     return result;
 }
 
-function linkForViolationsCategory(category: string, searchFilter: SearchFilter) {
+function linkForViolationsCategory(
+    category: string,
+    searchFilter: SearchFilter,
+    lifecycle: LifecycleOption,
+    hiddenSeverities: Set<PolicySeverity>
+) {
```

Comment:
In this case, distinct argument types enable TypeScript to report problems in function calls.

In cases when additional arguments are more generic, like string, then you might consider  argument object idiom so that property keys improve clarity and consistency between function definition and function call.